### DISCUSSION TITLE
Where is V8 development taking place?

### DISCUSSION THREAD

**Discussion by badsyntax** [react-hook-form/react-hook-form#8632] [Q&A] [answered]
Hi

I feel like an idiot, but I can't find any code for upcoming v8. Is it in a non-obvious branch?

**Comment by badsyntax** [comment]
Is it [react-18](https://github.com/react-hook-form/react-hook-form/tree/react-18)?

**Comment by Moshyfawn** [comment, marked as answer]
It's collective from multiple different branches as the changes of different aspects come in. `react-18` accumulates most of the changes, so.. yea, you can say that [react-18](https://github.com/react-hook-form/react-hook-form/tree/react-18) is the v8 branch.

@bluebill1049 correct me if I'm worng

**Reply by bluebill1049** [reply]
Yes, however, currently paused on the development on that branch as it encountered an issue with TS 4.7.,0 onwards.

**Reply by badsyntax** [reply]
Thanks both.
